aws-android-sdk-iot-test/src/androidTest/java/com/amazonaws/mobileconnectors/iot/MqttManagerIntegrationTest.java

```diff
@@ -31,88 +30,85 @@
 import com.amazonaws.services.iot.model.CreateKeysAndCertificateResult;
 import com.amazonaws.services.iot.model.CreatePolicyRequest;
-import com.amazonaws.services.iot.model.CreatePolicyResult;
 import com.amazonaws.services.iot.model.DeleteCertificateRequest;
 import com.amazonaws.services.iot.model.DeletePolicyRequest;
 import com.amazonaws.services.iot.model.DescribeEndpointRequest;
 import com.amazonaws.services.iot.model.DescribeEndpointResult;
 import com.amazonaws.services.iot.model.DetachPolicyRequest;
 import com.amazonaws.services.iot.model.ResourceAlreadyExistsException;
 import com.amazonaws.services.iot.model.UpdateCertificateRequest;
+import com.amazonaws.testutils.AWSTestBase;
 
 import org.json.JSONException;
-import org.json.JSONObject;
 import org.junit.After;
 import org.junit.Before;
-import org.junit.Ignore;
+import org.junit.BeforeClass;
 import org.junit.Test;
 
 import java.io.File;
 import java.security.KeyStore;
+import java.security.SecureRandom;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
-import java.util.Random;
+import java.util.Map;
 import java.util.concurrent.CountDownLatch;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
 
-public class MqttManagerIntegrationTest extends IoTIntegrationTestBase {
-
-    private static final Integer ONE_TWENTY_KB = 120000;
+public class MqttManagerIntegrationTest extends AWSTestBase {
+    private static final String TAG = MqttManagerIntegrationTest.class.getSimpleName();
 
-    private static String KEYSTORE_PATH = "./";
+    private static final int ONE_TWENTY_KB = 120_000;
     private static final String KEYSTORE_NAME = "integration_test.bks";
     private static final String KEYSTORE_PASSWORD = "test password";
     private static final String IOT_POLICY_NAME = "android-integration-test";
     private static final String CUSTOM_DOMAIN = "www.amplify-android-test.com";
 
-    private static boolean initCompleted = false;
-
+    private static String keyStorePath = "./";
     private static AWSCredentialsProvider credentialsProvider;
     private static String endpointPrefix;
     private static String betaEndpoint;
     private static String endpoint;
     private static AWSIotClient iotClient;
+    private static CreateKeysAndCertificateResult certResult;
 
-    private static Random r;
-    private static final String TAG = MqttManagerIntegrationTest.class.getSimpleName();
+    @BeforeClass
+    public static void beforeSuite() throws Exception {
+        Context appContext = InstrumentationRegistry.getTargetContext();
+        keyStorePath = appContext.getFilesDir().toString() + "/";
+        System.out.println(keyStorePath);
+        credentialsProvider = new CognitoCachingCredentialsProvider(
+            appContext,
+            getPackageConfigure("iot").getString(ConfigKey.IDENTITY_POOL_ID.toString()),
+            Regions.US_EAST_1
```

Comment:
Idle thought: I wonder if we should move all of these region declarations to the `testconfiguration.json`. Maybe someday later.

Reply:
I like it, I'll go for it.

Reply:
Probably not worth it but I defer to you!

Reply:
Eep, strike that. Ended up being a bigger deal than that. It isn't always us-east-1.